digiroad2-oracle/sql/insert_test_fixture.sql

```diff
@@ -5,4 +5,4 @@ Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_I
 Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (29,null,2,109,109,1140018963,1021227);
 Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (36,null,3,113,113,388554364,6488445);
 Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (37,null,3,113,113,388554364,6488445);
-Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (38,null,3,113,113,388554364,6488445);
+Insert into LRM_POSITION (ID,LANE_CODE,SIDE_CODE,START_MEASURE,END_MEASURE,MML_ID,LINK_ID) values (38,null,3,113,113,388554364,6488445);
```

Comment:
the id for this insert shouldn't ' be ` lrm_position_primary_key_seq.nextval ` ?

Reply:
In this case, the code didn't change. Git found some difference, like a space or enter